packages/lua-language-server/build.sh

```diff
@@ -0,0 +1,57 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/sumneko/lua-language-server"
+TERMUX_PKG_DESCRIPTION="Sumneko Lua Language Server coded in Lua"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@MrAdityaAlok <dev.aditya.alok@gmail.com>"
+TERMUX_PKG_VERSION=2.3.4
+TERMUX_PKG_SRCURL=https://github.com/sumneko/lua-language-server.git
+TERMUX_PKG_GIT_BRANCH="master"
```

Comment:
> How do I then clone submodules?

Aha, yeah, git submodules are not handled in a good way with source tar downloads, so git is the way to go then. 
Please change to `TERMUX_PKG_GIT_BRANCH=2.3.4` then instead

Reply:
Or even better, `TERMUX_PKG_GIT_BRANCH=$TERMUX_PKG_VERSION`

Reply:
I thought that but android patch has been added soon and it is still not released. So I would change as requested once it is released.

Reply:
Alright, please open a new PR to fix it when 2.3.5 or similar is out